server/src/main/java/io/crate/execution/dml/upsert/ShardUpsertRequest.java

```diff
@@ -483,11 +467,7 @@ public void writeTo(StreamOutput out, @Nullable Streamer[] insertValueStreamers)
             } else {
                 out.writeVInt(0);
             }
-            boolean sourceAvailable = source != null;
-            out.writeBoolean(sourceAvailable);
-            if (sourceAvailable) {
-                out.writeBytesReference(source);
-            }
+            out.writeBoolean(false);    // redundant source unavailable flag
```

Comment:
Could also add version checks, so that onOrAfter 5.8 we don't read/send the boolean anymore